io.github.Q1CHENL.fig.json

```diff
@@ -0,0 +1,112 @@
+  "command": "fig",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=xdg-run/gvfsd",
+    "--filesystem=home",
+    "--filesystem=xdg-pictures:rw",
+    "--filesystem=xdg-download:rw",
+    "--env=GDK_BACKEND=x11",
+    "--filesystem=xdg-run/pipewire-0"
+  ],
+  "build-options": {
+    "env": {
+      "PYTHON": "python3"
+    }
+  },
+  "modules": [
+    {
+      "name": "python3-setuptools",
+      "buildsystem": "simple",
+      "build-commands": [
+        "pip3 install . --prefix=${FLATPAK_DEST} --no-deps --ignore-installed"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://files.pythonhosted.org/packages/ed/22/a438e0caa4576f8c383fa4d35f1cc01655a46c75be358960d815bfbb12bd/setuptools-75.3.0.tar.gz",
+          "sha256": "fba5dd4d766e97be1b1681d98712680ae8f2f26d7881245f2ce9e40714f1a686"
+        }
+      ]
+    },
+    {
+      "name": "python3-pillow",
+      "buildsystem": "simple",
+      "build-commands": [
+        "pip3 install . --prefix=${FLATPAK_DEST} --no-deps --ignore-installed --no-build-isolation"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://files.pythonhosted.org/packages/a5/26/0d95c04c868f6bdb0c447e3ee2de5564411845e36a858cfd63766bc7b563/pillow-11.0.0.tar.gz",
+          "sha256": "72bacbaf24ac003fea9bff9837d1eedb6088758d41e100c1552930151f677739"
+        }
+      ]
+    },
+    {
+      "name": "pygobject",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dtests=false",
+        "-Dpython=/usr/bin/python3"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://files.pythonhosted.org/packages/2b/58/d34e67a79631177e3c08e7d02b5165147f590171f2cae6769502af5f7f7e/pygobject-3.50.0.tar.gz",
+          "sha256": "4500ad3dbf331773d8dedf7212544c999a76fc96b63a91b3dcac1e5925a1d103"
+        }
+      ]
+    },
+    {
+      "name": "libadwaita",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dexamples=false",
+        "-Dtests=false",
+        "-Dgtk_doc=false",
+        "-Dintrospection=enabled"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libadwaita/1.5/libadwaita-1.5.0.tar.xz",
+          "sha256": "fd92287df9bb95c963654fb6e70d3e082e2bcb37b147e0e3c905567167993783"
+        }
+      ]
+    },
+    {
+      "name": "fig",
+      "builddir": true,
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "dir",
+          "path": ".",
+          "include": ["style/*.css"]
+        }
+      ]
+    }
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "environment": {
+    "MESA_DEBUG": "silent",
+    "MESA_LOADER_DRIVER_OVERRIDE": "iris"
+  },
+  "metadata": {
+    "url": "https://github.com/Q1CHENL/fig"
+  }
```

Comment:
what part of the documentation say that both `environment` and `metadata` are valid?

So I can get it fixed.